core-tests/shared/src/test/scala/zio/stm/STMSpec.scala

```diff
@@ -187,28 +185,24 @@ final class STMSpec(implicit ee: org.specs2.concurrent.ExecutionEnv) extends Tes
       .repeat(Schedule.recurs(n) *> Schedule.identity)
 
   def e15 =
-    unsafeRun(
-      for {
-        tVar  <- TRef.makeCommit(0)
-        fiber <- ZIO.forkAll(List.fill(10)(incrementVarN(99, tVar)))
-        _     <- fiber.join
-        value <- tVar.get.commit
-      } yield value
-    ) must_=== 1000
+    for {
```

Comment:
Just to be consistent, can you remove `unsafeRun` in all the tests?

Reply:
Fixed 😆